lib/translations_checker/checker.rb

```diff
@@ -0,0 +1,93 @@
+require "translations_checker/diff"
+require "translations_checker/issues_presenter"
+require "translations_checker/terminal_colours"
+require "translations_checker/indentation"
+require "translations_checker/git_diff"
+require "translations_checker/change_checker"
+
+module TranslationsChecker
+  class Checker
+    include TerminalColours
+
+    using Indentation
+
+    def file_issues(en_diff)
```

Comment:
Can this method be private? I don't think it is used outside of this class.

Reply:
Done.